deps/build.jl

```diff
@@ -0,0 +1,28 @@
+using BinDeps
+using Compat
+
+@BinDeps.setup
+
+libmysqlclient = library_dependency("libmysqlclient",
+                                    alias=["libmysqlclient.so",
```

Comment:
Ah, you are right. It's `aliases` and not `alias`. It also turns out that WinRPM doesn't really help me in this case, because it only downloads dlls and does not download anything with which I can start the mysql server. So I have to download a `.zip` file from the website. 

So what's the best way of downloading a .zip file and then extract its contents to `deps/usr/` ?